### PR TITLE
Refactor with ES6 and Add Data & Time

### PR DESCRIPTION
**Pull request by Fahim-Khan-P**
### _**I have Added the following requirements to out last week Awesome Books group project!**_

- [ ] Use ES6 syntax.
- [ ] Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
- [ ] Keep all funcionalities of the app without errors.
- [ ] Refactor your methods and functions to arrow functions.
- [ ] Use let and const in a correct way.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+// const contactLi = document.getElementById('contact-li');
+// end of variable module
+
+// Class of book (seprate module)
+// class Books {
+
+//   constructor(title, author) {
+//     this.book = { id: this.allBooks.length + 1, title, author };
+//   }
+
```
Comment (christianbiring1) [inline index.js]: - [ ] Kindly consider removing all the comments lines of codes as they make your codebase looks ugly and it is not a good practice for clean code.

File: README.md [hunk 2]
```diff
@@ -1 +1,236 @@
...
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+```sh
+  git clone https://github.com/Fahim-Khan-P/awsome-book-ES6.git
+```
```
Comment (christianbiring1) [inline README.md]: - [ ] For making your README look more professional, you may consider providing the command on how to run the project locally after it has been cloned.
`Hint: cd <directory_name>`
`install third-party library: npm i`
`...`

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Fahim-Khan-P , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] The ES6 features were used `modules, arrow fx`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi  @Fahim-Khan-P 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * Use ES6 syntax. :white_check_mark: 
 * use new readme template provided by microverse :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions
- [ ]  N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**